dbt/include/global_project/macros/materializations/snapshots/snapshot.sql

```diff
@@ -30,8 +30,10 @@
   {% if not target_relation_exists %}
 
       {% set build_sql = build_snapshot_table(strategy, model['compiled_code']) %}
+      {{ log("--- build_sql: " ~ build_sql, info=true) }}
```

Comment:
Is this logging necessary to leave in, or was it just for debugging?

Reply:
Just an accident. Removed.